docker_db_config_template/8.18.1/EXAConf

```diff
@@ -104,8 +104,10 @@
     Affinity = 11
     UUID = F0EA64D47F374A00B0530772981C559EE1C59086
     DockerVolume = n11
+    # Commented out exposed ports as these are only used by
+    # https://github.com/exasol/docker-db/
     # Ports to be exposed (container : host)
-    ExposedPorts = 8888:8899, 6583:6594
+    # ExposedPorts = {{ db_port }}:8899, {{ bucketfs_port }}:6594
```

Comment:
I think, we removed this line in all other templates

Reply:
I thought, at least in the latest version this line could be useful, e.g. to remind the developer to remove this line when creating an `EXAConf` template for the next version.

Reply:
I think, if you diff the new ExaConf with old one, that we removed it

Reply:
removed.